index.js

```diff
@@ -37,42 +26,50 @@ const jsonParser = input => {
  * @property {boolean} [colorizeObjects=true] Apply coloring to rendered objects
  * when coloring is enabled.
  * @property {boolean} [crlf=false] End lines with `\r\n` instead of `\n`.
+ * @property {string|null} [customColors=null] A comma separated list of colors
```

Comment:
This doc block got reorganized to order the properties in alphabetical order. No real change here.